lib/post/utils/navigate_create_post.dart

```diff
@@ -68,7 +68,7 @@ Future<void> navigateToCreatePostPage(
                     l10n.postCreatedSuccessfully,
                     trailingIcon: Icons.remove_red_eye_rounded,
                     trailingAction: () {
-                      navigateToPost(context, postId: postViewMedia.postView.post.id);
+                      navigateToPost(context, postViewMedia: postViewMedia);
```

Comment:
Would it be worth removing the `postId` parameter from from `navigateToPost`? If the old post page needs it, maybe we can pass it farther along down the line.

Reply:
Hmm, I tried this and noticed that there are a couple of widgets that depend on the `postId`. Primarily, the modlog and the report page. I think we can keep this in for now since it would add a bit more logic to pre-fetch the post information!

Also, the plan is to re-introduce the `postId` for scenarios that require it (or don't have the full post view context)

Reply:
Got it!